mx.truffleruby/suite.py

```diff
@@ -297,6 +297,15 @@
               "TRUFFLERUBY_BOOTSTRAP_LAUNCHER": "<path:TRUFFLERUBY-BOOTSTRAP-LAUNCHER>/miniruby",
               "GRAALVM_TOOLCHAIN_CC": "<toolchainGetToolPath:native,CC>",
             },
+            "os_arch" : {
+                "linux-musl" : {
+                    "<others>" : {
+                        "buildEnv": {
+                            "NFI_LIB_ARG": "-L<path:truffle:TRUFFLE_NFI_NATIVE>/bin -ltrufflenfi",
```

Comment:
Do we need a `-I` too so `ffi.h` can be found reliably, even if there is no system-installed libffi?

Reply:
Added `-I` and another env var for headers. Tested on both linux-musl and linux-gnu without libffi-dev